src/output-filedata.c

```diff
@@ -167,6 +167,7 @@ static void OutputFiledataLogFfc(ThreadVars *tv, OutputLoggerThreadData *td,
                 SCLogDebug("ff FILE_STORE not set");
                 continue;
             }
+            DEBUG_VALIDATE_BUG_ON(ff->flags & FILE_NOSTORE);
```

Comment:
Argh this was kind of added in master with new `FileApplyTxFlags` checking
`DEBUG_VALIDATE_BUG_ON((file->flags & (FILE_STORE | FILE_NOSTORE)) == (FILE_STORE | FILE_NOSTORE));`

And S-V test hits this condition